src/Microsoft.AspNetCore.Server.IIS/Core/IISHttpContextOfT.cs

```diff
@@ -91,7 +93,7 @@ public override async Task<bool> ProcessRequestAsync()
                 }
 
                 // Cancel all remaining IO, there might be reads pending if not entire request body was sent by client
-                AsyncIO.Dispose();
+                AsyncIO?.Dispose();
```

Comment:
I don't see how `AsyncIO` can be null here.

Reply:
If connection was reset before response started